tests/files/axis/availability_with_checkouts_and_holds.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
None of our existing fixture files had checkouts and holds in them, so I added one from my testing.